server/controllers/assess/assessmentsController.ts

```diff
@@ -20,7 +21,7 @@ export default class AssessmentsController {
     return async (req: Request, res: Response) => {
       const assessment = await this.assessmentService.findAssessment(req.user.token, req.params.id)
 
-      if (assessment.status === 'pending') {
+      if (assessment.status === 'pending' && !informationSetAsNotReceived(assessment)) {
```

Comment:
This is making me wonder if it'd be better to have `informationReceived`, rather than needing the double-negative

Reply:
Yeah, I struggled with the naming here - we want to be sure it's actually set, and it also pushes an (arguably more confusing) double negative into the `getSections` method. Maybe I'll set this as a variable, so it's clearer as to what's going on.